.reviewdog.yml

```diff
@@ -0,0 +1,14 @@
+runner:
+  bandit:
+    cmd:  'bandit -r ./ -c .bandit.yaml -f custom --msg-template "{relpath}:{line}:{col}:{test_id}:{severity}:{msg}"' 
+    errorformat: # (optional if you use `format`)
+      - "%f:%l:%c:%m" 
+    name: bandit # (optional. you can overwrite <tool-name> defined by runner key)
+    level: info # (optional. same as -level flag. [info,warning,error])
+  # hadolint:
+  #   cmd: '/tmp/hadolint Dockerfile -f json | jq -f ".rdjson.jq" -c'
+  #   format: rdjson
+  #   level: error # (optional. same as -level flag. [info,warning,error])
```

Comment:
still need this? if not, remove it

Reply:
still need it. I am trying a different way to get reviewdog working. if it works then I can uncomment it and have it ran using reviewdog

Reply:
got it